gbfs.md

```diff
@@ -173,6 +175,43 @@ email             | Optional  | A single contact email address for customers to
 timezone          | Yes       | The time zone where the system is located. Time zone names never contain the space character but may contain an underscore. Please refer to the "TZ" value in https://en.wikipedia.org/wiki/List_of_tz_database_time_zones for a list of valid values
 license_url       | Optional  | A fully qualified URL of a page that defines the license terms for the GBFS data for this system, as well as any other license terms the system would like to define (including the use of corporate trademarks, etc)
 
+### vehicle_types.json
+The following fields are all attributes within the main "data" object for this feed.
+
+Field Name        | Required  | Defines
+------------------| --------- | ----------
+vehicle_types          | Yes       | Array that contains one object per vehicle type in the system as defined below
+\- vehicle_type_id      | Yes       | Unique identifier of a vehicle type. See [Field Definitions](#field-definitions) above for ID field requirements
+\- form_factor      | Yes       | An enumerable describing the vehicle's general form factor. <br /><br />Current valid values are:<br /><ul><li>`bicycle`</li><li>`scooter`</li><li>`car`</li></ul>
```

Comment:
I think there may be a missing `moped` value here for providers like Revel/Scoot.

Reply:
I went ahead and added `moped`. I also added the value `other` in order to allow the definition of any vehicle form factor that is not able to be categorized by any of the pre-defined form factors.